test/HybridCustody_tests.cdc

```diff
@@ -617,27 +616,38 @@ pub fun testGetNFTsAccessibleFromProxyAccount(){
 
     let expectedChildIDs = (scriptExecutor("example-nft/get_ids.cdc", [child.address]) as! [UInt64]?)!
     let expectedParentIDs: [UInt64] = []
-    let expectedAddressToIDs: {Address: [UInt64]} = {parent.address: expectedParentIDs, child.address: expectedChildIDs}
+    let expectedAddressToIDs: {Address: [UInt64]} = {child.address: expectedChildIDs, parent.address: expectedParentIDs}
 
     scriptExecutor(
         "test/test_get_nft_display_view_from_public.cdc",
         [parent.address, PublicPath(identifier: exampleNFTPublicIdentifier)!, expectedAddressToIDs]
     )
 
     // Test we have capabilities to access the minted NFTs
-    scriptExecutor("test/test_get_accessible_proxy_nfts.cdc", [parent.address, expectedAddressToIDs])
+    scriptExecutor("test/test_get_accessible_child_nfts.cdc", [
+        parent.address,
+        {child.address: expectedChildIDs} as! {Address: [UInt64]}
+    ])
     
-    // Mint new nfts from ExampleNFT2 and assert that get_accessible_proxy_nfts.cdc does not return these nfts.
+    // Mint new nfts from ExampleNFT2 and assert that get_accessible_child_nfts.cdc does not return these nfts.
     mintExampleNFT2Default(accounts[exampleNFT2]!, receiver: child)
     let expectedChildIDs2 = (scriptExecutor("example-nft-2/get_ids.cdc", [child.address]) as! [UInt64]?)!
-    let expectedAddressToIDs2: {Address: [UInt64]} = {parent.address: expectedParentIDs, child.address: expectedChildIDs2}
+    let expectedAddressToIDs2: {Address: [UInt64]} = {child.address: expectedChildIDs2, parent.address: expectedParentIDs}
 
     scriptExecutor(
         "test/test_get_nft_display_view_from_public.cdc",
         [parent.address, PublicPath(identifier: exampleNFT2PublicIdentifier)!, expectedAddressToIDs2]
     )
 
-    let error = expectScriptFailure("test/test_get_accessible_proxy_nfts.cdc", [parent.address, expectedAddressToIDs2])
+    // revoke the ExampleNFT2 provider capability, preventing it from being returned.
+    let paths: [CapabilityPath] = [
+        /private/exampleNFT2Collection,
+        /private/exampleNFTCollection
+    ]
+    txExecutor("misc/unlink_paths.cdc", [child], [paths], nil, nil)
+
+    let expectedAddressToIDsFails: {Address: [UInt64]} = {child.address: expectedChildIDs2}
+    let error = expectScriptFailure("test/test_get_accessible_child_nfts.cdc", [parent.address, expectedAddressToIDsFails])
```

Comment:
I'm a little confused by how this test ever passed, it never unlinked any providers. I did some adjustments here to bring it in-line with what I think the test is doing, but let's make sure it is still evaluating what we expect it to